### PR TITLE
Isvalidelement

### PR DESCRIPTION
**Pull request by KlonD90**
#21 implement isValidElement

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -84,6 +84,15 @@ var FastReactServer = {
         }
 
         return this.createElement.apply(this, newArgs);
+    },
+
+    /**
+     * @param {RenderElement} element
+     * @returns {boolean} isValid
```
Comment (sigorilla) [inline src/index.js]: I think it should be `Boolean` (in Title case)

File: src/index.js [hunk 2]
```diff
@@ -84,6 +84,15 @@ var FastReactServer = {
...
+     */
+    isValidElement: function (element) {
+        return element === null ||
+            (typeof element === 'object' && typeof element.type !== 'undefined' && typeof element.props === 'object');
```
Comment (KlonD90) [inline src/index.js]: 1. isObject equal typeof object == 'object'?
2. That's good.

File: src/index.js (hunk 1 above)
Comment (KlonD90) [inline src/index.js]: Ok. I think about it like a primitive, but it's a Boolean so that's right.

File: src/index.js (hunk 2 above)
Comment (alt-j) [inline src/index.js]: 1. yep

File: src/index.js [hunk 3]
```diff
@@ -84,6 +84,15 @@ var FastReactServer = {
         }
 
         return this.createElement.apply(this, newArgs);
+    },
+
+    /**
+     * @param {RenderElement} element
```
Comment (KlonD90) [inline src/index.js]: Hm?

Comment (alt-j) [inline src/index.js]: This method can take any type of argument.

File: src/index.js (hunk 2 above)
Comment (KlonD90) [inline src/index.js]: i write it now
```
    return object !== null && typeof object === 'object' && !Array.isArray(object);
```

**Comment by KlonD90** [comment]
Fix change requests. Please review ._.

File: src/index.js (hunk 3 above)
Comment (KlonD90) [inline src/index.js]: Get it.

**Comment by alt-j** [comment]
thanks for contribution 🎉

**Comment by KlonD90** [comment]
Thanks for release 🙆

**Pull request merged**